### PR TITLE
Add `fork_status` endpoint

### LINKED ISSUE #347: Improvement : add endpoint to return fork status
It could be useful to be able to request to devnet the fork info.
For example, with a such command :
`curl -X GET http://127.0.0.1:5050/fork-status -H "Content-Type:application/json"`
and have in return some infos :
- forked or not.
- source network of the fork (alpha-goerli, alpha-goerli2, alpha-mainnet, or URL).

### PR DESCRIPTION
**Pull request by ptisserand**
## Usage related changes

- Close #347 
  - Add `/fork_status` endpoint to return fork status

## Checklist:

- [x] Applied formatting - `./scripts/format.sh`
- [x] No linter errors - `./scripts/lint.sh`
- [x] Performed code self-review
- [x] Rebased to the last commit of the target branch (or merged it into my branch)
- [x] Documented the changes
- [x] Linked the issues which this PR resolves
- [x] Updated the tests
- [x] All tests are passing

### CODE REVIEW

File: starknet_devnet/blueprints/base.py [hunk 1]
```diff
@@ -189,3 +189,12 @@ async def create_block():
     """Create empty block"""
     block = await state.starknet_wrapper.create_empty_block()
     return Response(block.dumps(), status=200, mimetype="application/json")
+
+
+@base.route("/fork_status", methods=["GET"])
+async def fork_status():
+    """Get fork status"""
+    config = state.starknet_wrapper.config
+    if bool(config.fork_network):
```
Comment (FabijanC) [inline starknet_devnet/blueprints/base.py:198]: Unnecessary casting to `bool`

File: test/test_fork.py [hunk 2]
```diff
@@ -300,3 +302,34 @@ def test_deploy_account():
...
+    assert data.get("block") == TESTNET_DEPLOYMENT_BLOCK
+
+
+@devnet_in_background()
+def test_fork_status_not_forked():
+    """Test GET on /fork_status when not forking"""
+    resp = requests.get(f"{APP_URL}/fork_status")
+    assert resp.status_code == 200
+    data = resp.json()
+    assert len(data.keys()) == 0
```
Comment (FabijanC) [inline test/test_fork.py:335]: Can you instead assert that `data == {}`?

**Review by FabijanC — CHANGES_REQUESTED** [review]

File: starknet_devnet/blueprints/base.py (hunk 1 above)
Comment (ptisserand) [reply starknet_devnet/blueprints/base.py]: PR updated

**Review by FabijanC — APPROVED** [review]
Thanks for the contribution!

**Pull request merged**